utils/gyb_syntax_support/StmtNodes.py

```diff
@@ -326,4 +326,18 @@
                    is_optional=True),
              Child('Body', kind='CodeBlock'),
          ]),
+
+    # e.g. #assert(1 == 2)
+    Node('PoundAssertStmt', kind='Stmt',
+         children=[
+             Child('PoundAssert', kind='PoundAssertToken'),
+             Child('LeftParen', kind='LeftParenToken'),
+             Child('Condition', kind='Expr',
+                   description='The assertion condition.'),
+             Child('Comma', kind='CommaToken', is_optional=True,
+                   description='The comma after the assertion condition.'),
+             Child('Message', kind='StringLiteralExpr', is_optional=True,
```

Comment:
`StringLiteralExpr ` -> `StringLiteralToken`.
This is why the last test case in `test/Syntax/round_trip_parse_gen.swift` isn't wrapped with `<PoundAssertStmt />`

Reply:
oops, didn't notice that. thanks for catching. fixed!